### DISCUSSION TITLE
Pi 4 w/ SX1302 SPI (CoreCell), any luck?

### DISCUSSION THREAD

**Discussion by Citizen-2CB8A24A** [meshtastic/firmware#3025] [Q&A] [answered]
I'd like to create a Meshtastic station (mature gateway w/ roof antenna), just for fun and to see how things go. I like the mesh idea and having plenty gateways around seem to be more than welcome. ;-)

Last days I've played with a SENSECAP M1-8868 Helium mining LoRaWAN Model, but skipped the Helium thing and went directly into the TTN. But it's not much fun, if you haven't any sensors though. ;-) And there's no mesh anyways.
At present, there's a LILYGO T-ECHO underway, arriving early next year. The idea is now, to operate the T-ECHO in the car and the former TTN gateway at the house.
While dropping Helium, installing PiOS and the Docker based Basicstation software for TTN integration was very easy (kinda turn-key solution), I'm somewhat lost with Meshtastic. For answers I tried to join Discord first, as stated in the FAQs, but they want my real phone number for registering with them. No, I won't give them my private phone number, just for having it leaked at one day. Now I'm here and hoping for the best.

I haven't found any hints regarding my CEEED card (having that SX1302 SPI chip) in the devices area on the website so far.

Are there any plans to support those outstanding (almost turn-key) gateways (Pi4 w/ 8GB RAM) at one day? Those boxes are quite cheap to buy (used and new ones) since many Helium operators seem to give up spare units or blacklisted ones.

Thank you!

**Comment by jp-bennett** [comment, marked as answer]
That is something we would love to support. Unfortunately, Meshtastic uses RadioLib to interface with the radio, and RadioLib has no real support for the SX1302 at this time. For now it's just not possible.
